sysutils/ndcctools/Portfile

```diff
@@ -3,39 +3,43 @@
 PortSystem              1.0
 
 name                    ndcctools
-version                 6.2.10
+version                 7.6.1
 revision                0
-checksums               rmd160  38c802fa436ea33ace1ce420e83653d862262b1d \
-                        sha256  cd97639ad58b2a72cccd2b16f231d3fef7dfb65d6b63d9141e578f9598a6ba76 \
-                        size    12804180
+checksums               rmd160 29e910251b99ee02377bf711922641adb95cca01 \
+                        sha256  6892eb22b5765e8b3b134997a1e0d6462ade6271a6a42a8a14b4ab30eff8893c \
+                        size    14064797
 
 categories              sysutils
-platforms               darwin
-license                 MIT
+platforms               darwin \
+                        linux
+license                 GPL-2
 
 maintainers             {nd.edu:dthain @dthain} \
+                        {nd.edu:btovar @btovar} \
                         openmaintainer
 
-set description_common  {Notre Dame Cooperative Computing Software}
-description             ${description_common} (cooperative-computing-lab)
-long_description        ${description_common} \
-                        Enables collaborators to easily harness large scale \
-                        distributed systems such as clusters, clouds, and grids.
+description             distributed computing tools for scientific workflows
+long_description        ${name} is a toolkit that provides a variety of workflow \
+                        and task execution systems for executing complex scientific \
+                        workflows on clusters, clouds, and grids.
  
-homepage                http://ccl.cse.nd.edu/
+homepage                https://cctools.readthedocs.io
 master_sites            http://ccl.cse.nd.edu/software/files/
 distname                cctools-${version}-source
 
+depends_lib             port:python311 \
+                        port:openssl \
+                        port:readline \
+                        port:curl \
+                        port:swig \
+                        port:swig-python
+
```

Comment:
are all the dependencies specified as the correct type? Oftentimes, thing like "swig" or "swig-python" are only build dependencies. If you specify them as "depends_lib" that implies build- and runtime dependencies. Can you please verify this?

Reply:
Good point, I have updated swig to depends-build